src/components/CurrencySelectionList/index.tsx

```diff
@@ -28,23 +38,56 @@ function CurrencySelectionList({searchInputLabel, initiallySelectedCurrencyCode,
             return acc;
         }, [] as CurrencyListItem[]);
 
+        const policyRecentlyUsedCurrencyOptions: CurrencyListItem[] =
+            recentlyUsedCurrencies?.map((currencyCode) => {
+                const currencyInfo = currencyList?.[currencyCode];
+                const isSelectedCurrency = currencyCode === initiallySelectedCurrencyCode;
+                return {
+                    currencyName: currencyInfo?.name ?? '',
+                    text: `${currencyCode} - ${CurrencyUtils.getCurrencySymbol(currencyCode)}`,
+                    currencyCode,
+                    keyForList: currencyCode,
+                    isSelected: isSelectedCurrency,
+                };
+            }) ?? [];
```

Comment:
```suggestion
            Array.isArray(recentlyUsedCurrencies) ? recentlyUsedCurrencies?.map((currencyCode) => {
                const currencyInfo = currencyList?.[currencyCode];
                const isSelectedCurrency = currencyCode === initiallySelectedCurrencyCode;
                return {
                    currencyName: currencyInfo?.name ?? '',
                    text: `${currencyCode} - ${CurrencyUtils.getCurrencySymbol(currencyCode)}`,
                    currencyCode,
                    keyForList: currencyCode,
                    isSelected: isSelectedCurrency,
                };
            }) : [];
```

We can probably do something like this for now, https://expensify.slack.com/archives/C02NK2DQWUX/p1725615472542989?thread_ts=1724690852.626429&cid=C02NK2DQWUX